e2e/wdio/core/tests/feed-input.e2e-spec.ts

```diff
@@ -0,0 +1,248 @@
+import {
+    addIsActiveClass,
+    addValue, checkElementScreenshot,
+    clearValue, click,
+    doesItExist,
+    elementDisplayed,
+    executeScriptAfterTagFF,
+    getAttributeByName,
+    getAttributeByNameArr,
+    getCSSPropertyByName,
+    getElementArrayLength,
+    getElementSize,
+    isEnabled, mouseHoverElement,
+    refreshPage, saveElementScreenshot,
+    scrollIntoView,
+    sendKeys,
+    setValue,
+    waitForElDisplayed,
+    waitForPresent
+} from '../../driver/wdio';
+import { FeedInputPo } from '../pages/feed-input.po';
+import { four_lines_text, eight_lines_text} from '../fixtures/testData/feed-input';
+
+import {
+    activeInputTextAreasHoverState,
+    disableInputButtonsExample,
+    disableInputButtonsHoverState,
+    disableInputTextAreasExample,
+    disableInputTextAreasHoverState,
+    activeInputTextAreasExample,
+    activeInputButtonsExample,
+    activeInputButtonsHoverState,
+    activeInputTextAreasFocusState,
+    activeInputButtonsFocusState,
+    activeInputButtonsActiveState
+} from '../fixtures/testData/feed-input-tags';
+
+import {
+    placeholder_array, default_avatar, send_button_tooltip,
+    button, textArea
+} from '../fixtures/appData/feed-input-page-contents';
+
+describe('Verify Feed Input component', function() {
+    const {
+        feedInputTextArea, feedInputAvatar, feedInputButton, feedInputNoAvatar, disableInputTextArea,
+        disableInputButton, activeInputTextAreas
+    } = new FeedInputPo();
+    const feedInputPage = new FeedInputPo();
+
+    beforeAll(() => {
+        feedInputPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForElDisplayed(feedInputAvatar);
+    }, 1);
+
+    it('should have correct placeholder assigned', () => {
+        const feedInputLength = getElementArrayLength(feedInputTextArea);
+        for (let i = 0; i < feedInputLength; i++) {
+            expect(getAttributeByNameArr(feedInputTextArea, 'placeholder')).toEqual(placeholder_array);
```

Comment:
this only checks index 0, need to do like this:
expect(getAttributeByNAmeArr(feedInputTextArea, 'placeholder', i)).toEqual(placeholder_array[i]);